### ISSUE TITLE
Degree profiles - course overview accordion tabs have incorrect `role`

### ISSUE THREAD

**Issue by cjg89** [UCF/Main-Site-Theme#392]
**Describe the bug**

See snippet below--because `.accordion` has a role of `tablist`, it must contain at least one `tab` element.  The `<a>` tags that act as tabs in this component have `role="button"` instead:
https://github.com/UCF/Main-Site-Theme/blob/9c234610ac63589c9311722579c592cacc41d04b/template-parts/degree/course_overview.php#L20-L22

**Steps to Reproduce**

n/a

**Expected Behavior**

`.accordion` should contain a list of `tab` role elements.

**Possible Fix**

Update the `role` attribute in the snippet above in the `course_overview.php` template part, and confirm this snippet no longer fails "required owned elements" checks.

**Your environment:**

n/a

**Additional context**

n/a
